src/app/projects/components/project-settings/components/labeling-tasks/labeling-tasks.component.html

```diff
@@ -0,0 +1,408 @@
+                                    Task Type</th>
+                                <th scope="col"
+                                    class="px-3 py-2 text-center text-xs font-medium uppercase tracking-wide text-gray-500">
+                                    Labels</th>
+                                <th scope="col"
+                                    class="px-3 py-2 text-center text-xs font-medium uppercase tracking-wide text-gray-500">
+                                </th>
+                            </tr>
+                        </thead>
+                        <tbody class="divide-y divide-gray-200">
+                            <tr *ngFor="let labelingTaskContainer of labelingTasksArray.controls; let j = index"
+                                [ngClass]="j%2!=0 ? 'bg-gray-50':'bg-white'" [formGroupName]="j">
+                                <td class="whitespace-nowrap text-center px-3 py-2 text-sm text-gray-500">
+                                    {{labelingTaskContainer.get("targetName").value}}
+                                </td>
+                                <td class="whitespace-nowrap text-center px-3 py-2 text-sm text-gray-500">
+                                    <p class="break-words cursor-pointer"
+                                        [ngClass]="labelingTaskContainer.get('nameOpen').value?'hidden':null"
+                                        (click)="openTaskName(labelingTaskContainer)">
+                                        {{labelingTaskContainer.get("name").value}}</p>
+                                    <ng-template [ngIf]="labelingTaskContainer.get('nameOpen').value">
+                                        <input #inputTaskName name="labelName" type="text" placeholder="Task Name"
+                                            [value]="labelingTaskContainer.get('name').value"
+                                            class="w-52 input input-outline input-bordered input-sm"
+                                            (keyup)="checkTaskNameColor($event.target)"
+                                            (keydown.enter)="changeTaskName(labelingTaskContainer,inputTaskName.value)"
+                                            (blur)="changeTaskName(labelingTaskContainer,inputTaskName.value)">
+                                    </ng-template>
+                                </td>
+                                <td class="whitespace-nowrap text-center px-3 py-2 text-sm text-gray-500">
+                                    <select formControlName="taskType"
+                                        class="select select-sm select-bordered pr-8 max-w-xs">
+                                        <option *ngFor="let dropdown of labelingTasksDropdownValues()"
+                                            [attr.disabled]="isLabelingTaskOptionDisabled(labelingTaskContainer,dropdown.value)?true:null"
```

Comment:
function

Reply:
I am resolving the conversation because it will be solved by other issue and the function in the html will not exist